### ISSUE TITLE
Built image not runnable

### ISSUE THREAD

**Issue by shinigambit** [buildpacks/docs#499]
**On Page:** [Make your application runnable](https://buildpacks.io/docs/buildpack-author-guide/create-buildpack/make-app-runnable/) the produced image after running the tutorial buildpack fails to start with the following message: `bundle exec ruby app.rb: bundle exec ruby app.rb: command not found`

**Comment by AidanDelaney** [comment]
Thanks for the report @shinigambit.  Could I ask which version of `pack` you are using?  The output of `pack report` would also be helpful to debug this.

**Comment by shinigambit** [comment]
Hi @AidanDelaney the version I'm using is 0.27.0, the output of `pack report` is:
```Pack:
  Version:  0.27.0
  OS/Arch:  linux/amd64

Default Lifecycle Version:  0.14.1

Supported Platform APIs:  0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9

Config:
  default-builder-image = "[REDACTED]"

  [[trusted-builders]]
    name = "[REDACTED]"
```
I don't know if it's relevant but I followed the tutorial inside a WSL2 Ubuntu instance

**Comment by AidanDelaney** [comment]
Can we connect on Slack to debug this issue?

**Comment by AidanDelaney** [comment]
I think our fix to #518 addresses this. Please re-open if it hasn't.
